installing/index.adoc

```diff
@@ -25,7 +25,7 @@ include::modules/ipi-verifying-nodes-after-installation.adoc[leveloffset=+2]
 
 * xref:../post_installation_configuration/bare-metal-configuration.adoc#getting-the-baremetalhost-resource_post-install-bare-metal-configuration[Getting the BareMetalHost resource]
 
-* xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#ipi-install-troubleshooting-following-the-installation_ipi-install-installation-workflow[Following the installation]
+* xref:../installing/installing_bare_metal_ipi/ipi-install-installation.adoc#ipi-install-troubleshooting-following-the-installation_ipi-install-installation[Following the installation]
```

Comment:
You may consider going with something like "Checking the installation progress" or "Tracking the installation progress" instead of "Following the installation". I think that "following" could lead to some confusion given that it could be interpreted or mistranslated as "next steps". Just a thought though, not a serious merge blocker.

Reply:
I'm going to leave it for now, because we are going to consolidate and restructure both the IPI and UPI docs under a different JIRA ticket.